oak_functions_launcher/src/main.rs

```diff
@@ -43,7 +43,7 @@ mod server;
 #[derive(clap::Subcommand, Clone, Debug, PartialEq)]
 enum Mode {
     /// Launch enclave in crosvm
-    Crosvm(crosvm::Params),
+    Virtual(virtualized::Params),
```

Comment:
What's the difference between Virtual and Virtualized?
Which one is the launch mode?

Reply:
I really wanted to name everything `Virtual`/`virtual`, but `virtual` is a reserved keyword.

Launch mode is `Virtual`.